src/shared/GitHub/GitHubConstants.cs

```diff
@@ -51,13 +58,19 @@ public static class OAuthScopes
         public static class EnvironmentVariables
         {
             public const string AuthenticationModes = "GCM_GITHUB_AUTHMODES";
+            public const string OAuthClientId = "GCM_GITHUB_CLIENTID";
```

Comment:
The cover letter says that these are for testing only and not officially supported.
I'm wondering if we want to encode that in the env var name.  Git uses GIT_TEST_*
for test-only vars and GIT_* for supported vars, for example.

Reply:
That’s a good idea. I like GCM to try and stay consistent with git practices.